api/main.py

```diff
@@ -83,4 +83,21 @@ async def get_restaurants():
 @app.get("/neighborhoods", response_model=List[NeighborhoodModel])
 async def get_neighborhoods():
     neighborhoods = await db["neighborhoods"].find().to_list(100)
+
     return neighborhoods
+
+
+@app.get("/neighborhood", response_model=NeighborhoodModel)
```

Comment:
Returns one neighborhood closest to the `lon` and `lat` query parameters

http://localhost:3000/neighborhood?lat=-73.961704&lon=40.662942